seaworthy/tests-core/test_container_base.py

```diff
@@ -3,7 +3,7 @@
 from docker.models.containers import Container
```

Comment:
`test_container*s*_base.py`?

Reply:
I went back and forth a few times on that. If you have a preference, I'm happy to change to match.